package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@1hive/1hive-ui",
-  "version": "1.0.16",
+  "version": "1.1.0",
```

Comment:
It just changed the types, should up to 1.1.0 or 1.1.17 what you think @0xGabi?

Reply:
Let's use `1.1.17`

Reply:
I typo we want say `1.0.17` right?

Reply:
Yes, published already!